metaparser/androidsignature/signature_test.go

```diff
@@ -22,30 +22,46 @@ func Test_ReadAABSignature(t *testing.T) {
 
 	gitCommand, err := git.New(tmpDir)
 	require.NoError(t, err)
-	err = gitCommand.Clone("https://github.com/bitrise-io/sample-artifacts.git").Run()
+	err = gitCommand.Clone("https://github.com/bitrise-io/sample-artifacts.git", "-b", "update-aabs").Run()
```

Comment:
Fix after https://github.com/bitrise-io/sample-artifacts/pull/26 merged.

Reply:
Fixed in https://github.com/bitrise-io/go-android/pull/64/commits/c3059aa4427c023f98b42ded35af62488fa31103